src/subtask.ts

```diff
@@ -14,7 +14,12 @@ export class ULabelSubtask {
         active_id: string,
         annotation_mode: string,
         back_context: CanvasRenderingContext2D,
-        edit_candidate: unknown, // TODO: figure out what type this is
+        edit_candidate: {
+            annid: string,
+            access: ULabelAnnotation,
+            distance: number,
+            point: ULabelSpatialPayload  // TODO: double check this one 
```

Comment:
this is the `mouse_location`, or a set of coordinates `[global_x, global_y]`, which is indeed a point. `ULabalSpatialPayload` is a list of points. We don't have a dedicated class for Points yet, but should eventually (#106 ). In `geometric_utils.ts` we do have `Point2D`, which if we wanna use elsewhere we should move to `index.d.ts` (#131), so we can do that or change this to just `[number, number]` 
```suggestion
            point: [number, number] 
```